CHANGES

```diff
@@ -6,6 +6,7 @@ Bugs fixed
 
 * #11618: Fix a regression in the MoveModuleTargets transform,
   introduced in #10478 (#9662).
+* #11649: Use :py:func:`calendar.timegm` to convert from UTC time to UNIX epoch.
```

Comment:
```suggestion
* #11649: linkcheck: Fix conversions from UTC to UNIX time
  for timezones west of London.
```

Reply:
The conversion was wrong for timezones east of London, too. Just the test didn't hang there.